examples/langchain/simple_chain.py

```diff
@@ -0,0 +1,19 @@
+import os
+import mlflow
+
+from langchain.chains import LLMChain
+from langchain.prompts import PromptTemplate
+from langchain.llms import OpenAI
+
+llm = OpenAI(temperature=0.9)
```

Comment:
This example will not work w/o OpenAI API key. We should at least set that to a value like `environ["OPENAI_API_KEY"] = "FIXME"`